src/conn/conn_api.c

```diff
@@ -1139,6 +1143,10 @@ __conn_close(WT_CONNECTION *wt_conn, const char *config)
             WT_TRET(__wt_session_close_internal(s));
         }
 
+    if (conn->default_session->event_handler) {
```

Comment:
The external sessions are closed before doing the rollback_to_stable call at shutdown. I wanted to capture stats for that as well, so I let the application know a connection is available again for use.